Content.Server/Body/Systems/RespiratorSystem.cs

```diff
@@ -189,6 +189,22 @@ private void StopSuffocation(EntityUid uid, RespiratorComponent respirator)
             respirator.Saturation =
                 Math.Clamp(respirator.Saturation, respirator.MinSaturation, respirator.MaxSaturation);
         }
+
+        private void OnApplyMetabolicMultiplier(EntityUid uid, RespiratorComponent component, ApplyMetabolicMultiplierEvent args)
+        {
+            if (args.Apply)
+            {
+                component.CycleDelay *= args.Multiplier;
+                component.Saturation *= args.Multiplier;
```

Comment:
why saturation?

Reply:
You start suffocating otherwise

Reply:
It's reduced by cycledelay each tick per the docs so they need to be scaled together
![image](https://user-images.githubusercontent.com/60792108/157348662-6533e121-f417-474b-aba0-263bef10c371.png)

Reply:
yeah but you arent scaling max or min saturation so this is going to break shit